spec/controllers/information_controller_spec.rb

```diff
@@ -2,6 +2,10 @@
 require 'rails_helper'
 
 RSpec.describe InformationController, type: :controller do
+  before do
+    user = FactoryGirl.create(:user) # add here
```

Comment:
You will need to call user on its own, or use let!, possibly you haven't created when you call it.